Solution: TacitRed Defender Threat Intelligence (Official)

mazamizo21 opened the pull request:
## Official Data443 Submission
This is the official submission from the Data443 organization for the **TacitRed Defender Threat Intelligence** solution.

### Changes
- Standardized publisher information to 'Data443 Risk Mitigation, Inc.'.
- Added comprehensive documentation.
- Validated package metadata.

This PR supersedes and replaces PR #13247.
Please close #13247 in favor of this one.

mazamizo21 commented:
## Proactive Update: Common Issues Pre-Fixed

Hi Microsoft Team,

Based on feedback patterns from our other PRs (#13267, #13268, #13269, #13278), we have proactively addressed the following items:

### ✅ 1. Moved packageMetadata.json outside Package folder
- **Before**: `Solutions/TacitRed-Defender-ThreatIntelligence/Package/packageMetadata.json`
- **After**: `Solutions/TacitRed-Defender-ThreatIntelligence/packageMetadata.json`

### ✅ 2. Created Images folder in Playbooks with running playbook screenshots
- `Solutions/TacitRed-Defender-ThreatIntelligence/Playbooks/TacitRedToDefenderTI/Images/TacitRedToDefenderTILight.png`
- `Solutions/TacitRed-Defender-ThreatIntelligence/Playbooks/TacitRedToDefenderTI/Images/TacitRedToDefenderTIDark.png`

### ✅ 3. Created ReleaseNotes.md in standard table format

Please let us know if any additional changes are required.

Thank you!

**Data443 Risk Mitigation, Inc.**

v-maheshbh commented:
Hi @mazamizo21 
We are putting your PR on hold because your solution uses Azure Function and not Sentinel Codeless Connector Framework (CCF). We would be happy to work with you to migrate your solution to CCF. Please contact the Microsoft Sentinel Partners at [AzureSentinelPartner@microsoft.com] (mail to: AzureSentinelPartner@microsoft.com) to discuss available options.

Thanks!

v-maheshbh commented:
> Hi @mazamizo21
> 
> Kindly verify why the playbook is not visible in the Content Hub.
> 
> <img alt="image" width="675" height="402" src="https://private-user-images.githubusercontent.com/207855009/536714585-24388db0-ce7f-43a4-9c01-141a63d66a7d.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njg5ODc4MjIsIm5iZiI6MTc2ODk4NzUyMiwicGF0aCI6Ii8yMDc4NTUwMDkvNTM2NzE0NTg1LTI0Mzg4ZGIwLWNlN2YtNDNhNC05YzAxLTE0MWE2M2Q2NmE3ZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTIxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDEyMVQwOTI1MjJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04YTEwYWQwODE2YzkxZWFhYWUyZjE3ZDE2Zjc5YTNlOTk3NzdkYzI3OWM5OTFhMWNiMzA5MjY1Mzk0MDU2OTY4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.dtYsyTvZWY9tHB-5BbcmvAZzSU2NYDBIv0V9JR6-9NE">
> Thanks!

Hi @mazamizo21 
Please review an existing solution for reference and make the necessary updates in the playbook.

Thanks!

mazamizo21 commented:
## Publisher ID Update

### Changes Made
- **SolutionMetadata.json**: Updated `publisherId` from `data443` to `data443riskmitigationinc1761580347231`
- **mainTemplate.json**: Updated `solutionId` to `data443riskmitigationinc1761580347231.azure-sentinel-solution-tacitred-defender-ti`
- **V3 package regenerated**: `mainTemplate.json` and `3.0.0.zip` updated

### Partner Center Alignment
This update aligns the package with the Partner Center publisher ID. When creating the Partner Center offer, use:
- **Offer ID**: `azure-sentinel-solution-tacitred-defender-ti`

### Validation
- V3 packaging: ✅ Passed
- ARM-TTK: ✅ All tests passed
- Post-V3 validation: ✅ No null values, no deprecated links

Commit: `3921edced2`

v-maheshbh commented:
Hi @mazamizo21 

<img width="1328" height="432" alt="image" src="https://github.com/user-attachments/assets/76cb0ba4-afce-4846-bc6a-d21791713445" />

I have made some modifications to the playbook, and it is now visible in the Content Hub. However, I am facing access issues while deploying the **Azure Function**. Kindly deploy it from your end and attach the screenshot once completed.

Thanks!
